src/components/business/DeleteIdentity.tsx

```diff
@@ -35,11 +36,25 @@ export const DeleteIdentity: React.FC<DeleteIdentityProps> = ({
     value: value,
   };
 
+  const deleteMessage = `${
+    type.toUpperCase() === IdentityTypes.EMAIL
```

Comment:
You should use `getIdentityType` utils functions instead of `toUpperCase`, that way we keep the infered type.

Reply:
OK fixed a653af4